specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "serviceName": "sampleServiceName",
+	"serviceMemberId": "SampleServiceMemberId",
+    "api-version": "2014-01-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "connectors": [
+              {
+                "connectorId": "Id of the connector",
```

Comment:
Can you give an example of this field which looks like a real-life connectorId? Is this an ARM resourceId?

Reply:
It is not an ARM resource ID, it is just a guid that identifies a AAD Connect Sync connector.

Reply:
Here is an example response.   "connectors": [
    {
      "connectorId": "3efbb72d-5047-4816-a7dd-73fefe16a2f3",
      "connectorDisplayName": "adhsdfus1runner.adhs.com"
    },
    {
      "connectorId": "b891884f-051e-4a83-95af-2544101c9083",
      "connectorDisplayName": "dfadhybridhealth.ccsctp.net - AAD"
    }
  ],
  "runProfileNames": [
    "Export",
    "ApplyRules",
    "DeltaImport"
  ]

Reply:
Thanks! I meant to ask - can you update the PR with this? It's always useful if the examples show data which looks like real-life data.